src/RestClient/endpoints/V3/Customers/CustomerConsent.ts

```diff
@@ -0,0 +1,34 @@
+import { AxiosInstance } from 'axios';
+
+import { getCustomersPath } from './Customers';
+import { CustomerConsent } from './types';
+
+class CustomersConsent {
+  private client: AxiosInstance;
+
+  constructor(client: AxiosInstance) {
+    this.client = client;
+  }
+
+  /**
+   * Gets the status of a single customer's consent
+   *
+   * @param customerId A valid Customer ID
+   * @returns Promise resolving to single customer consent status
+   */
+  get(customerId: string): CustomerConsent['GetResponse'] {
```

Comment:
For consistency across this library, please require that all ID's passed into HTTP methods are of type `number` instead of `string`. See: https://github.com/bigcommerce/bigcommerce-api-node/blob/31c2a95746c67ff2eaeb735123d45d0fdfc79228/src/RestClient/endpoints/V3/Catalog/Products.ts#L76 as an example 👍

Reply:
Got it! Added that change in as well. @matthewvolk